smart_control/environment/environment.py

```diff
@@ -1308,7 +1298,7 @@ def _action_strings(
 
     # Exit when the episode has ended and return terminal step information.
     # We still need to get the final observation to add to the transition.
-    self._episode_ended = self._has_episode_ended(last_timestamp)
```

Comment:
removing unused arg `last_timestamp` (https://github.com/google/sbsim/pull/32/files#r2070815640) from the caller